debian/changelog

```diff
@@ -1,3 +1,9 @@
+golang-logrus (0.7.1-1) experimental; urgency=medium
```

Comment:
This should be `s/experimental/UNRELEASED/` until it's actually being pushed to the archive (and whoever does the pushing does the replacement).